<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-CVT4PTN" failures="1" tests="2" name="Test" time="25.461" errors="0" timestamp="2025-09-22T20:19:04 IST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="5.882"/>
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="11.624">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC002_LoginTest.verify_login(TC002_LoginTest.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
</testsuite> <!-- Test -->
